src/util/findLast.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+
+// This is simply a reverse-order version of Array.prototype.find()
+// Returns the last element in the array that satisfies the provided
+// testing function.
+// Otherwise, it returns undefined, indicating that no element passed the test.
+export function findLast<T>(
+  array: $ReadOnlyArray<T>,
+  testingFunction: (obj: T) => boolean
+): ?T {
+  let index = array.length - 1;
+  while (index >= 0 && !testingFunction(array[index])) index--;
+  if (index >= 0) return array[index];
```

Comment:
Would be nice if findLast composed findLastIndex

Reply:
Concurred

Reply:
Accepted and updated.